spec/data-spec.js

```diff
@@ -185,6 +189,13 @@ describe("Data", () => {
 				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
 				expect(+circle.attr("cy")).to.be.closeTo(expectedCy[556][i], 1);
 			});
+
+			d3SelectAll(".bb-circles-778-889 .bb-circle").each(function (d, i) {
+				const circle = d3Select(this);
+
+				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
+				expect(+circle.attr("cy")).to.be.closeTo(expectedCy["778.889"][i], 0);
```

Comment:
Yeah, i'd love to, cannot get karma to run correctly, i get dozens of failed tests even on clean clone from master.
thank god for github squash